8.js

```diff
@@ -0,0 +1,168 @@
+const real = new Float32Array(2)
+const imag = new Float32Array(2)
+const audioContext = new AudioContext()
+const oscillator1 = audioContext.createOscillator()
```

Comment:
I haven't scrolled past this yet, but whenever I see variables named `thing1` through `thingN`, I wonder if an array might be a more suitable representation?